tests/test_encoder.py

```diff
@@ -129,8 +129,31 @@ def test_encode_feature_with_id(self):
         self.assertRoundTrip(input_geometry=geometry,
                              expected_geometry=[[1, 1]], id=42)
 
-    def test_encode_multipolygon(self):
-        geometry = 'MULTIPOLYGON (((40 40, 20 45, 45 30, 40 40)), ((20 35, 10 30, 10 10, 30 5, 45 20, 20 35), (30 20, 20 15, 20 25, 30 20)))'  # noqa
+    def test_encode_multilinestring(self):
+        geometry = 'MULTILINESTRING ((10 10, 20 20, 10 40), (40 40, 30 30, 40 20, 30 10))'  # noqa
         self.assertRoundTrip(input_geometry=geometry,
-                             expected_geometry=[[40, 40], [20, 45], [45, 30], [40, 40]],  # noqa
-                             expected_len=2)
+                             expected_geometry=[
+                                 [[10, 10], [20, 20], [10, 40]],
+                                 [[40, 40], [30, 30], [40, 20], [30, 10]],
+                             ])
+
+    def test_encode_multipolygon_normal_winding_order(self):
+        geometry = 'MULTIPOLYGON (((40 40, 20 45, 45 30, 40 40)), ((20 35, 10 30, 10 10, 30 5, 45 20, 20 35), (30 20, 20 15, 20 25, 30 20)))'  # noqa
+        self.assertRoundTrip(
+            input_geometry=geometry,
+            expected_geometry=[
```

Comment:
It looks like the MultiPolygon passed as input has 2 outers and 1 inner on the second outer. The returned array seems to have 3 outers?